origin.spec

```diff
@@ -0,0 +1,419 @@
+#debuginfo not supported with Go
+%global debug_package %{nil}
+%global gopath      %{_datadir}/gocode
+%global import_path github.com/openshift/origin
+%global kube_plugin_path /usr/libexec/kubernetes/kubelet-plugins/net/exec/redhat~openshift-ovs-subnet
+%global sdn_import_path github.com/openshift/openshift-sdn
+
+# docker_version is the version of docker requires by packages
+%global docker_verison 1.6.2
+# tuned_version is the version of tuned requires by packages
+%global tuned_version  2.3
+# openvswitch_version is the version of openvswitch requires by packages
+%global openvswitch_version 2.3.1
+# %commit and %ldflags are intended to be set by tito custom builders provided
+# in the rel-eng directory. The values in this spec file will not be kept up to date.
+%{!?commit:
+%global commit 86b5e46426ba828f49195af21c56f7c6674b48f7
+}
+%global shortcommit %(c=%{commit}; echo ${c:0:7})
+# ldflags from hack/common.sh os::build:ldflags
+%{!?ldflags:
+%global ldflags -X github.com/openshift/origin/pkg/version.majorFromGit 0 -X github.com/openshift/origin/pkg/version.minorFromGit 0+ -X github.com/openshift/origin/pkg/version.versionFromGit v0.0.1 -X github.com/openshift/origin/pkg/version.commitFromGit 86b5e46 -X k8s.io/kubernetes/pkg/version.gitCommit 6241a21 -X k8s.io/kubernetes/pkg/version.gitVersion v0.11.0-330-g6241a21
+}
+
+%if "%{dist}" == ".el7ose"
+%global package_name atomic-openshift
+%global product_name Atomic OpenShift
+%else
+%global package_name origin
+%global product_name Origin
+%endif
+
+Name:           %{package_name}
+# Version is not kept up to date and is intended to be set by tito custom
+# builders provided in the rel-eng directory of this project
+Version:        0.0.1
+Release:        0%{?dist}
+Summary:        Open Source Container Management by Red Hat
+License:        ASL 2.0
+URL:            https://%{import_path}
+ExclusiveArch:  x86_64
+Source0:        https://%{import_path}/archive/%{commit}/%{name}-%{version}.tar.gz
+
+BuildRequires:  systemd
+BuildRequires:  golang >= 1.4
+
+
+%description
+%{summary}
+
+%package master
+Summary:        %{product_name} Master
+Requires:       %{name} = %{version}-%{release}
+Requires(post): systemd
+Requires(preun): systemd
+Requires(postun): systemd
+
+%description master
+%{summary}
+
+%package node
+Summary:        %{product_name} Node
+Requires:       %{name} = %{version}-%{release}
+Requires:       docker-io >= %{docker_version}
```

Comment:
Yes, it's slightly more complicated than just requiring 1.6.2 due to bugs in later packages but we definitely require a minimum of 1.6.2 for core functionality to work.